google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcConversions.java

```diff
@@ -518,6 +525,22 @@ private Bucket.IamConfig.UniformBucketLevelAccess ublaEncode(BucketInfo.IamConfi
     return to.build();
   }
 
+  private BucketInfo.Autoclass autoclassDecode(Bucket.Autoclass from) {
+    BucketInfo.Autoclass.Builder to = BucketInfo.Autoclass.newBuilder();
+    to.setEnabled(from.getEnabled());
+    ifNonNull(from.getToggleTime(), timestampCodec::decode, to::setToggleTime);
+    return to.build();
+  }
+
+  private Bucket.Autoclass autoclassEncode(BucketInfo.Autoclass from) {
+    Bucket.Autoclass.Builder to = Bucket.Autoclass.newBuilder();
+    if (from.getEnabled() != null) {
+      to.setEnabled(from.getEnabled());
+    }
```

Comment:
nit: this can use ifNonNull as well
```suggestion
    ifNonNull(from.getEnabled(), to::setEnabled);
```